Update the game over message (#1)
jadengee123/skills-review-pull-requests: README.md

</header>

<!--
  <<< Author notes: Step 5 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

> ## Step 5: Apply suggested changes

Author: ```suggestion
## Step 4: Apply suggested changes
```

</header>

<!--
  <<< Author notes: Step 4 >>>
  Start this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

## Step 4: Suggest changes

> _Nice work reviewing that pull request :sparkles:_

Author: ```suggestion
_Nice work reviewing that pull request :sparkles:_
```

Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.

> **What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.

Author: ```suggestion
**What is _suggest changes_?**: Thieature enables you to recommend a change to a pull request that the author can commit with the push of a button.
```
